komodo/build.py

```diff
@@ -42,7 +42,7 @@ def dfs(pkg, version, pkgs, repo):
     return flatten(dependencies)
 
 
-def rpm(pkg, ver, path, data, prefix, *args, **kwargs):
+def rpm(pkg, ver, path, prefix):
```

Comment:
Big chage, do you know for sure nothing is calling this function with positional args, or passing `*args` or `**kwargs`?

Reply:
It should only be called from the make() function, but that is an assumption  

Reply:
After chatting, I think I get it.